programs/librapay_api/src/librapay_instruction.rs

```diff
@@ -36,39 +31,39 @@ pub fn mint(
     let ix_data = LoaderInstruction::InvokeMain { data };
 
     let accounts = vec![
-        AccountMeta::new_readonly(*program_pubkey, false),
-        AccountMeta::new(*from_pubkey, true),
-        AccountMeta::new(*to_pubkey, false),
+        AccountMeta::new_readonly(*script, false),
+        AccountMeta::new(*genesis, true),
+        AccountMeta::new(*to, false),
     ];
 
     Instruction::new(solana_sdk::move_loader::id(), &ix_data, accounts)
 }
 
 pub fn transfer(
-    program_pubkey: &Pubkey,
-    mint_pubkey: &Pubkey,
-    from_pubkey: &Pubkey,
-    to_pubkey: &Pubkey,
+    script: &Pubkey,
+    genesis: &Pubkey,
+    from: &Pubkey,
```

Comment:
Why remove the `_pubkey` suffixes?

Reply:
Redundant and noisy, reminds me of:
```
enum Car {
  FordCar,
  ChevyCar,
};
```

Reply:
It's not redundant and I regret writing the original programs in exactly the form you've changed this code back too. It's a step backwards.  Your `script` variable, for example, doesn't hold a script.  Please use the form `<qualifier>_<noun>`, where `<noun>` is typically the type.  That's the way most of the rest of the codebase is written.

Reply:
Yeah, but that's the point, `script` does not hold a script but it does represent the script and you can easily see its a pubkey without having to drag that extra noise around.  In long complex functions where there are multiple forms of "script" (its bytes, its keypair, its pubkey, etc...) adding the `<noun>` makes is helpful.  But as a broad rule, it just makes things unnecessarily noisy imho.  Looking around the code there is a combination of both styles. If there is a consensus that all variables will add a `<noun>` from now on then I don't have any strong objections and will change these back.

Reply:
So by the "when in Rome..." philosophy, I think we should add the `<noun>`. The noise is better than the inconsistency.